pom.xml

```diff
@@ -43,27 +41,42 @@
         <dependency>
           <groupId>com.amazonaws</groupId>
           <artifactId>dynamodb-streams-kinesis-adapter</artifactId>
-          <version>${dynamodb-streams-kinesis-adapter.version}</version>
```

Comment:
As a convention I'm only using properties if there's more than one dependency using that version.

Reply:
I removed the "not yet supported" message altogether, since the class itself is sufficiently descriptive.